tables/Tables/ArrayColumn.h

```diff
@@ -501,6 +504,45 @@ class ArrayColumn : public TableColumn
     mutable Bool reaskAccessColumnSlice_p;
 };
 
```

Comment:
I'd like some comments in class ColumnSlicer. Furthermore, a test program seems to be lacking.
Furthermore, I wonder why ColumnSlicer has a Vector<Slicer*>. It raises the question why it is not <const Slicer*>, but also: why not Vector<Slicer>? Do you think that constructing it is too expensive?